test/test_tensor.py

```diff
@@ -190,5 +190,14 @@ def test_element_size(self):
       dtype = f.default
       assert dtype.itemsize == Tensor.randn(3, dtype=dtype).element_size(), f"Tensor.element_size() not matching Tensor.dtype.itemsize for {dtype}"
 
+  def test_cumsum(self):
```

Comment:
Should be in test_ops. It's okay if multidimensional doesn't work with images, but you have to skip them appropriately